handler/category-dimension-import.go

```diff
@@ -188,14 +191,27 @@ func (h *CategoryDimensionImport) Handle(ctx context.Context, e *event.CategoryD
 	return nil
 }
 
-// instanceFailed updates the instance state to 'failed' and returns an Error wrapping the original error and any error during the state update
-func (h *CategoryDimensionImport) instanceFailed(ctx context.Context, err error, e *event.CategoryDimensionImport) error {
-	if _, err1 := h.datasets.PutInstanceState(ctx, h.cfg.ServiceAuthToken, e.InstanceID, dataset.StateFailed, headers.IfMatchAnyETag); err1 != nil {
-		return &Error{
+// failed updates the instance and the import states to 'failed' and returns an Error wrapping the original error and any error during the state update
+func (h *CategoryDimensionImport) failed(ctx context.Context, err error, e *event.CategoryDimensionImport) error {
```

Comment:
Would it be better if this function was called something like:
setImportToFailed() ?